themes/admin/Sea_Green/css/sea_green.css

```diff
@@ -20094,4 +20094,9 @@ table [class^=ri-], table [class*=" ri-"] {
 
 .ls-card-grid .ls-card-grid__description {
   padding-left: 8px;
-}
\ No newline at end of file
+}
+
+*:focus{
+  outline: 2px solid currentColor !important;
+}
```

Comment:
this generates a double line on certian buttons like the purple create survey, also this is the build file and should not be modified here but inside the assets/admin_themes/Sea_Green then build through gulp with gulp build_theme or gulp watch_theme